js/index.js

```diff
@@ -1 +1,74 @@
 // Your code goes here
+
+const logoHeading = document.querySelector('h1');
+console.log(logoHeading);
+logoHeading.addEventListener('mouseover', event => {   
+event.target.style.color = 'yellow';
+setTimeout(function() {
+    event.target.style.color = "";
+  }, 1500);
+}, false);
+
+
+const buttons = document.querySelectorAll('.btn');
+buttons.forEach(button =>{
+button.addEventListener('click',event =>{
+event.target.style.backgroundColor = 'yellow';
+});
+});
+
+
+
+const dottedLines = document.getElementsByClassName('intro');
+// let dottedLines = document.getElementsByClassName('content-section inverse-content');
+// let dottedLines = document.getElementsByClassName('content-pickgit');
+dottedlines.addEventListener('scroll', event => {
+event.target.style.borderBottom = '2px dashed black';
+});
+
+
+const key = document.getElementsByClassName('container home');
+key.addEventListener('keyword', event => {
+event.window.alert('WELCOME TO THE FUN BUS!!!');
+});
+
+
+const exclamation = document.querySelector('header h2');
+exclamation.addEventListener('compositionupdate', event =>{
+event.textContent = '!';
+});
+
+
+const busPic = document.querySelector('header img');
+busPic.addEventListener('load', event =>{
+event.style.border = '1px black';
```

Comment:
```suggestion
event.target.style.border = '1px black';
```
You're trying add a style to the event object instead use event.targer